kimchi/src/circuits/polynomials/keccak/mod.rs

```diff
@@ -0,0 +1,103 @@
+//! Keccak hash module
+pub mod circuitgates;
+pub mod gadget;
+
+pub const DIM: usize = 5;
+pub const QUARTERS: usize = 4;
+pub const ROUNDS: usize = 24;
+pub const RATE: usize = 1088 / 8;
```

Comment:
Nit: I like having the unit in the variable name. For instance, `RATE_IN_BYTES`. It helps when you use it somewhere else. When you expose it, it creates a better API.